acceptance-tests/src/test/java/tech/pegasys/eth2signer/dsl/lotus/LotusNode.java

```diff
@@ -33,20 +29,14 @@
  * TEXLOTUSDEVNET_BIGSECTORS=false -p 1234:7777 \ -v /tmp/import:/tmp/import textile/lotus-devnet
  * </code>
  */
-public class LotusNode {
-  private static final String FC_URL_FORMAT = "http://%s:%d/rpc/v0";
+public class LotusNode extends FilecoinJsonRpcEndpoint {
+  private static final String FC_URL_FORMAT = "http://%s:%d";
 
   private final String fcUrl;
-  private final JsonRpcClient jsonRpcClient;
-  public static final ObjectMapper OBJECT_MAPPER =
-      new ObjectMapper().registerModule(new FilecoinJsonRpcModule());
 
   public LotusNode(final String host, final int port) {
+    super(new ObjectMapper().registerModule(new FilecoinJsonRpcModule()), "/rpc/v0");
```

Comment:
nit: you can potentially create ObjectMapper as final state in this class (or in the base class), and expose it as a getter so that other tests can use the same instance. I believe both Lotus and Signer meant to use FilecoinJsonRPCModule, correct?

Reply:
Yeah, I hadn't thought of a good way to handle this - definitely should be only 1 of these mappers.

Reply:
I suspect it should jus be created in the FilecoinJsonRpcEndpoint directly